pywr/recorders/_recorders.pyx

```diff
@@ -1022,6 +1022,33 @@ cdef class TotalFlowNodeRecorder(BaseConstantNodeRecorder):
 TotalFlowNodeRecorder.register()
 
 
+cdef class MeanFlowNodeRecorder(BaseConstantNodeRecorder):
+    """
+    Record the mean flow for a Node.
+
+    A factor can be provided to scale the total flow (e.g. for calculating operational costs).
+    """
+    def __init__(self, *args, **kwargs):
+        self.factor = kwargs.pop('factor', 1.0)
+        super(MeanFlowNodeRecorder, self).__init__(*args, **kwargs)
+
+    cpdef after(self):
+        cdef ScenarioIndex scenario_index
+        cdef int i
+        cdef int days = self.model.timestepper.current.days
```

Comment:
doesn't look like the `days` variable is used anywhere

Reply:
Fixed.